### DISCUSSION TITLE
Material property to set color

### DISCUSSION THREAD

**Discussion by Futuremappermydud** [ToniMacaroni/SaberFactory#93] [Ideas]
Add a optional property for materials that determines which color to use, if it doesn’t exist then use default

**Comment by ToniMacaroni** [comment]
isn't this what the `custom color` switch already does?
If it's off then the color in your material won't be overridden.

**Reply by Futuremappermydud** [reply]
I mean to define which saber color to use

**Reply by Futuremappermydud** [reply]
So that right saber color can go on a material in left saber
